transform/snowflake-dbt/tests/data_quality/hightouch/subscriptions/test_cloud_professional.sql

```diff
@@ -1,11 +1,11 @@
 {{ config(
-    tags = ['data-quality']
+    tags = ["data-quality", "deprecated"]
 ) }}
 
 SELECT
     *
 FROM
-    {{ ref('dq_cloud_professional') }}
+    {{ ref("dq_cloud_professional') }}
```

Comment:
"dq_cloud_professional' -> "dq_cloud_professional'" ?

Reply:
Nice catch, not sure why it was changed. Some IDE dark magic. Will keep the original `'`.